test/plugin_test/plugin_test.go

```diff
@@ -172,6 +172,8 @@ var _ = Describe("Service Manager Plugins", func() {
 			{"bind", "PUT", "/v2/service_instances/1234/service_bindings/111"},
 			{"unbind", "DELETE", "/v2/service_instances/1234/service_bindings/111"},
 			{"fetchBinding", "GET", "/v2/service_instances/1234/service_bindings/111"},
+			{"pollInstance", "GET","/v2/service_instances/1234/last_operation"},
```

Comment:
should we verify the plugin execution is triggered when query parameters are used

Reply:
Good point... This is an issue for all endpoints currently. Not sure how to handle that. It would be easiest to forward all calls `/v1/osb/{brokerId}/**` regardless of whether they're OSB. After all, a broker might implement its own API that's not OSB but still can be called by someone